website/docs/MigrationV7.md

```diff
@@ -0,0 +1,95 @@
+---
+id: migration-v7
+title: Migration to 7.0
+---
+
+:::caution
+We renamed the `react-native-testing-library` npm package to `@testing-library/react-native`, joining the Testing Library family.
+:::
+
+This guide consists of two parts, because it involves merging two libraries together into a single one. Please choose the one that's relevant to you:
+
+- [guide for `react-native-testing-library` users](#guide-for-react-native-testing-library-users)
+- [guide for `@testing-library/react-native` users](#guide-for-testing-libraryreact-native-users)
+
+# Guide for `react-native-testing-library` users
+
+This guide describes steps necessary to migrate from React Native Testing Library `v2.x` to `v7.0`.
+
+## Renaming the library
+
+1. Install `@testing-library/react-native`.
+1. Uninstall `react-native-testing-library`.
+1. Rename all references of `react-native-testing-library` to `@testing-library/react-native`.
+
+You may have noticed a strange v2 to v7 upgrade, skipping versions 3, 4, 5 and 6. This is because we renamed the `react-native-testing-library` npm package to `@testing-library/react-native`, officially joining the "Testing Library" family 🎉. We're merging existing two libraries into a single one. The [native-testing-library](https://github.com/testing-library/native-testing-library/issues) repository will soon be archived and using `@testing-library/react-native` below v7, sourced from mentioned repository, is deprecated.
```

Comment:
Removing `/issues` from the repo link.

```suggestion
You may have noticed a strange v2 to v7 upgrade, skipping versions 3, 4, 5 and 6. This is because we renamed the `react-native-testing-library` npm package to `@testing-library/react-native`, officially joining the "Testing Library" family 🎉. We're merging existing two libraries into a single one. The [native-testing-library](https://github.com/testing-library/native-testing-library) repository will soon be archived and using `@testing-library/react-native` below v7, sourced from mentioned repository, is deprecated.
```